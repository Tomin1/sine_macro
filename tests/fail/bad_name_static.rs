/*
 * Copyright (c) 2025 Tomi Lepp√§nen
 * SPDX-License-Identifier: MIT
 */

use sine_macro::sine_wave;

sine_wave! {
    static STATIC_WAVE = something_else();
}

fn main() {}
