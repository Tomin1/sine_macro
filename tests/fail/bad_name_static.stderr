error: the identifier must be `sine_wave`
 --> tests/fail/bad_name_static.rs:9:26
  |
9 |     static STATIC_WAVE = something_else();
  |                          ^^^^^^^^^^^^^^
